index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="stylesheets/styles.css">
+  <link rel="stylesheet" href="font-awesome-4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css"
+  href="https://fonts.googleapis.com/css?family=Lobster|Harmattan|Josefin+Sans|Abel|Lato|Playfair+Display">
+  <title>Haby Randall</title>
+</head>
+<body id="page1">
+  <header>
+    <div class="header-name">
+      <a href="about.html" target="_blank">Haby Randall</a>
+    </div>
+    <nav class="top-nav">
+      <ul>
+        <li class="btn"> <a href="hobby-blog.html" target="_blank">Hobbies</a></li>
+        <li class="btn"> <a href="porfolio.html" target="_blank">Porfolio</a></li>
+        <li class="btn"> <a href="about.html">About Me</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class="introduction">
+      <article class="intro-paragraph">
```

Comment:
looks like your articles have two different classes: "intro-paragraph" and "about-me-paragraph" yet the same styling. So these could be under the same class, but since the classes weren't called in your stylesheet, I wouldn't give them a class at all. If this was this was something you did to label/keep track of your paragraphs, I would use comments instead.